test/integration/spec/localparticipant.js

```diff
@@ -1394,4 +1394,48 @@ describe('LocalParticipant', function() {
       assert(!error);
     });
   });
+
+  describe.only('JSDK-2985 - Event Listeners cleaned up after leaving room', () => {
```

Comment:
Please remove `.only`  - I have a precomit hooks that would help avoid this. @manjeshbhargav - is it okay to merge https://github.com/twilio/twilio-video.js/pull/1069 ?

Reply:
@makarandp0 👍  as long as it's not enabled by default.